weberm/sunshineEmr/views.py

```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+def index(request):
+    data = {"content": "Gfg is the best"}
+    return render(request, "index.html", data)
```

Comment:
@crespi1998 here it is. You might have looked at an old version of it. There is a lot of files so i understand